frontend/src/components/userPage/UserPageContent.jsx

```diff
@@ -316,6 +361,66 @@ const UserPageContent = ({ user }) => {
                             {user.username} Data:
                         </p>
                     </div>
+                    {user.cellphoneNumber != null && (
+                        <div>
+                            <div className="mb-4">
+                                <label className="block text-gray-800 text-sm font-bold mb-2">
+                                    Cellphone number:
+                                </label>
+                                <input
+                                    type="tel"
+                                    className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                                    value={cellphoneNumber ?? ''}
+                                    placeholder="(00) 00 00000000"
+                                    onChange={(e) => {
+                                        setCellphoneNumber(e.target.value)
+                                    }}
+                                    required
+                                />
+                            </div>
+                            <button
+                                className="bg-blue-700 hover:bg-blue-950 text-white font-bold py-1 px-2 rounded mr-[1rem] mb-[.5rem]"
+                                type="button"
+                                onClick={handlePatchCellphone}
+                            >
+                                Change cellphone number
+                            </button>
+                            <button
+                                className="bg-red-700 hover:bg-blue-950 text-white font-bold py-1 px-2 rounded mr-[1rem] mb-[.5rem]"
+                                type="button"
+                                onClick={handleDeleteCellphone}
+                            >
+                                Delete cellphone number
+                            </button>
```

Comment:
You can allow the user to pass in an empty cellphone number in change cellphone number. It will make the user flow more simplistic

Reply:
Done